Missing statuses explained in the basics (#30)
wealthkernel/api-documentation-guides: docs/inbound-transfers/Basics.md

 Accepted | The inbound transfer has been accepted by the ceding provider. 
 Transferring | The funds for the inbound transfer have been sent by the ceding provider to the portfolio managed by WealthKernel. 
 Completed | The funds for the inbound transfer have been received and are available in the portfolio managed by WealthKernel.

>  Rejected | The inbound transfer has been rejected by WealthKernel. This could happen if your porfolio or account didn´t pass validation checks done by WealthKernel. The reason will be communicated through the status change history.

Reviewer: ```suggestion
 Rejected | The inbound transfer has been rejected by WealthKernel. The reason will be communicated in the status history entry.
```

>  Cancelled | The inbound tranfer has been cancelled by WealthKernel. This state is irreversible.

Reviewer: Two things, remember that we didn't cancel it, we just approved it, and also the fact that it's irreversible should be clear to be honest. I know we put it in the UI, but I don't think it's necessary here.
```suggestion
 Cancelled | The inbound transfer has been cancelled.
```
